lib/helpers/api.dart

```diff
@@ -202,25 +203,13 @@ class KnockoutAPI {
     }
   }
 
-  Future<List<SubforumThreadLatestPopular>> latestThreads() async {
-    final response = await _request(type: 'get', url: 'thread/latest');
-
+  Future<List<SignificantThread>> getSignificantThreads(
+      SignificantThreads threadsToFetch) async {
+    final endpoint = "v2/threads/${threadsToFetch.name.toLowerCase()}";
```

Comment:
There's a version 2 out, maybe we should slowly convert each endpoint to this version before just adding `v2/` to the end of `KNOCKOUT_URL` ?

Reply:
![image](https://user-images.githubusercontent.com/8688472/118185905-5eb72e00-b43d-11eb-88d3-27007df630db.png)
aha aha

Reply:
We convert as we go. 👍